src/docs/perf/rendering/shader.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Reduce shader compilation jank on mobile
+short-title: Shader jank
+description: What is shader jank and how to minimize it.
+---
+
+If the animations on your mobile app appear to be janky,
+but only on the first run, you can _warm up_ the
+shader for the Skia Shader Language (SkSL) for a
+significant improvement.
+
+![Side-by-side screenshots of janky mobile app next to non-janky app]({% asset perf/render/shader-jank.gif @path %}){:width="100%"}
```

Comment:
The `{:width="100%"}` looks bad in my browser as the GIF is upscaled to full width which looks very blurry on my high-resolution screen. Shall we keep the GIF to be rendered at the original resolution?

Reply:
I removed it for the first image. The tracing image was HUGE and extended beyond the window, so I sized it down.